pkg/api/routers/router.go

```diff
@@ -61,6 +61,7 @@ func init() {
 			beego.NSRouter("/:tenantId/pools", &controllers.PoolPortal{}, "get:ListPools"),
 			beego.NSRouter("/:tenantId/pools/:poolId", &controllers.PoolPortal{}, "get:GetPool"),
 			beego.NSRouter("/:tenantId/availabilityZones", &controllers.PoolPortal{}, "get:ListAvailabilityZones"),
+			beego.NSRouter("/:tenantId/metrics", controllers.NewMetricsPortal(), "get:GetMetrics"),
```

Comment:
I would suggest to move metrics API to another new file `metric.go`, please refer to [alert.go](https://github.com/opensds/opensds/blob/development/pkg/api/routers/alert.go).

Reply:
Thanks, I have moved to a new file metric.go